### DISCUSSION TITLE
Misspell: 'Noticible' on faq page?

### DISCUSSION THREAD

**Discussion by wxqdoit** [wevm/viem#1381] [General]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Package Version

1.16.0

### Current Behavior

Not sure if it's a typo?
In https://viem.sh/docs/faq.html,
Sentence: We do not use the term "Signer" because there are noticible behavioral differences between signing locally and signing over JSON-RPC.
Word: noticible

### Expected Behavior

Sentence: We do not use the term "Signer" because there are noticeable behavioral differences between signing locally and signing over JSON-RPC.
Word: noticeable

**Comment by tmm** [comment]
Feel free to create a PR!
